tests/UI/config.dist.js

```diff
@@ -38,12 +38,18 @@ exports.mocha = 'mocha-3.1.2';
 exports.chai = 'chai-1.9.0';
 
 /**
- * The mocha reporter to use.
+ * Mocha reporters to use (can be multiple delimited by a comma).
  */
-exports.reporter = "spec";
+exports.reporter = 'mocha-multi-reporters';
+exports.reporterOptions = {
+  reporterEnabled: 'spec, @testomatio/reporter/lib/adapter/mocha.js',
+  testomatioReporterLibAdapterMochaJsReporterOptions: {
+    apiKey: process.env.TESTOMATIO
+  }
+};
```

Comment:
Shouldn't we only set this if `process.env.TESTOMATIO` has a value. Otherwise all plugin tests or even when running tests locally will try to use the TESTOMATIO reporter (with an empty api key).

Reply:
Added condition if `process.env.TESTOMATIO` has value, run testomat reporting, else run regular spec. Thanks @sgiehl